docs/security/trust/content_trust.md

```diff
@@ -152,10 +152,21 @@ To enable content trust in a `bash` shell enter the following command:
 export DOCKER_CONTENT_TRUST=1
 ```
 
-Once set, each of the "tag" operations require key for trusted tag. All of these
-commands also support the `--disable-content-trust` flag. This flag allows
-publishers to run individual operations on tagged images without content trust on an
-as-needed basis. 
+Once set, each of the "tag" operations require key for trusted tag. 
```

Comment:
Hm, for *a* trusted tag?

TL;DR I think there's something missing in the sentence, LOL